pom.xml

```diff
@@ -850,6 +840,52 @@
                 </plugins>
             </build>
         </profile>
+        <profile>
+            <id>all-modules</id>
+            <modules>
+                <module>messaging</module>
```

Comment:
It's best to leave the modules that are not dependent on a profile to be defined outside of the <profiles> tags, in the generic part of the pom.

Reply:
I already had my doubts, the java 17 one is in a separate one now, and only added for java 17.